View/TableWithPlot/FdaDataGrid.cs

```diff
@@ -448,10 +449,10 @@ private void myDataGrid_AutoGeneratingColumn(object sender, DataGridAutoGenerati
                             if (dna != null)
                             {
                                 Console.WriteLine(dna.DisplayName);
-                                if(dna.DisplayIndex >= 0)
-                                {
-                                    dgtc.DisplayIndex = dna.DisplayIndex;
-                                }
+                                //if(dna.DisplayIndex >= 0)
```

Comment:
if we'd like to keep commented out code, it would be helpful to include some comments on why we've left the commented out code in the class.

Reply:
agreed. gone.